backend/src/personnel/dto/bcws/create-bcws-personnel.dto.ts

```diff
@@ -171,4 +171,20 @@ export class CreatePersonnelBcwsDTO {
     required: false,
   })
   division: number;
+
+  @ApiProperty({
+    description: 'First Choice Section',
+    required: false,
+    enum: Section,
+    example: Section.AVIATION,
+  })
+  firstChoiceSection: Section;
```

Comment:
I think these should be optional?

Reply:
(I know they aren't required but sometimes I get an error when I don't put the '?: string' on the field)

Reply:
Will merge, can change in the future if it causes an issue